stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/ChatDomainImplTest.kt

```diff
@@ -79,7 +79,7 @@ internal class ChatDomainImplTest {
             val awaitingAttachmentsMessage = randomMessage(
                 syncStatus = SyncStatus.AWAITING_ATTACHMENTS,
                 attachments = mutableListOf(
-                    randomAttachment { uploadState = Attachment.UploadState.InProgress },
+                    randomAttachment { uploadState = Attachment.UploadState.InProgress(10) },
```

Comment:
Is this `10` on purpose?

Reply:
Just for testing, I tried to use some middle value instead of 0 or 100 to ensure if there is any weird behaviour.